carbon-design-system/carbon-addons-iot-react
# feat(card): add new datetimepicker to card as option

Closes #2454

**Summary**

- Adds a new option to the Card availableActions that will render the custom range picker

**Change List (commits, features, bugs, etc)**

- feat(Card): allow non-boolean values in the available actions
- feat(CardToolbar): add new timepicker if a availableActions.range is of type string and take additional time range options
- feat(cardToolbar): smaller font inside cards
- feat(CardPropTypes): extend the existing props for timeRange and timeRangeOptions and availableActions.range
- fix(TimeSeriesCard): was generating sample values even in the case where we weren't in editable mode

**Acceptance Test (how to verify the PR)**

- Verify new card range picker stories send the action correctly

**Regression Test (how to make sure this PR doesn't break old functionality)**

- Verify old card range picker still works in the story.



**Things to look for during review**
